### PR TITLE
STY: Use meta.labels

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses pysat/pysat#587

Updates instruments to use the new meta.labels functionality.  This will not pass on Travis until the pysat branch is merged, but it is passing locally.

## Type of change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Using `pytest` with the `meta_reshape` branch of `pysat`.

**Test Configuration**:
* Mac OS X 10.15.7
* Python 3.8.2

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes (part of "style update" line from previous pull).

### CODE REVIEW

File: pysatMissions/instruments/pysat_ephem.py [hunk 1]
```diff
@@ -186,31 +186,29 @@ def load(fnames, tag=None, inst_id=None, obs_long=0., obs_lat=0., obs_alt=0.,
...
+meta['alt'] = {meta.labels.units: 'km',
+               meta.labels.desc: "WGS84 height above Earth's surface"}
+meta['position_ecef_x'] = {meta.labels.units: 'km',
+                           meta.labels.desc: 'ECEF x co-ordinate of satellite'}
+meta['position_ecef_y'] = {meta.labels.units: 'km',
+                           meta.labels.desc: 'ECEF y co-ordinate of satellite'}
+meta['position_ecef_z'] = {meta.labels.units: 'km',
+                           meta.labels.desc: 'ECEF z co-ordinate of satellite'}
+meta['obs_sat_az_angle'] = \
+    {meta.labels.units: 'degrees',
```
Comment (aburrell) [inline pysatMissions/instruments/pysat_ephem.py:207]:
```suggestion
meta['obs_sat_az_angle'] = {
    meta.labels.units: 'degrees',
```

Here and elsewhere.

**Review by aburrell — APPROVED** [review]
I made a style suggestion that I think makes the code easier to read, but I don't remember if it's a part of our currently adopted standards so I'm approving.

File: pysatMissions/instruments/pysat_ephem.py (hunk 1 above)
Comment (jklenzing) [reply pysatMissions/instruments/pysat_ephem.py]: I like this.  Pushed an update.  Should I leave this unmerged until the main main branch is merged?

Comment (aburrell) [reply pysatMissions/instruments/pysat_ephem.py]: Yes, I'm planning on re-triggering the tests on all these pulls after merging the meta one.

**Pull request merged**